msfdb

```diff
@@ -1097,6 +987,19 @@ if $PROGRAM_NAME == __FILE__
 
   parse_args(ARGV)
 
+  update_db_port
+
+  if @standalone
+    @db_driver = Standalone.new(options: @options, msf_pass: @msf_pass, msftest_pass: @msftest_pass, db_conf: @db_conf)
+  elsif installed?("pg_ctl") && has_requirements(PgCtl.requirements)
+    @db_driver = PgCtl.new(db_path: @db, options: @options, localconf: @localconf, db_conf: @db_conf)
+  elsif installed?("pg_ctlcluster") && has_requirements(PgCtlcluster.requirements)
+    @db_driver = PgCtlcluster.new(db_path: @db, options: @options, localconf: @localconf, db_conf: @db_conf)
+  else
+    abort
```

Comment:
The fall through case here isn't entirely clear, I think before it warned you that you needed particular requirements in order to run msfdb - but now I think it just aborts? 👀